src/components/MessageCenter/NotificationDrawer.tsx

```diff
@@ -19,6 +9,14 @@ import {
 } from '../BoundingClientAwareComponent/BoundingClientAwareComponent';
 import { style } from 'typestyle';
 
+// KIALI-3172 For some reason not fully explained, when loaded with "import" it happens that the NotificationDrawer
+// does not come with the expected ".Title", ".Accordion" (etc.) fields.
```

Comment:
Could you import those explicitly?

Reply:
You mean, the composed parts (.Title, .Accordion etc) ?
This is what @israel-hdez did here https://github.com/kiali/kiali-ui/pull/1339
But (as far as I understand) the problem is that it's pointing explicitly to the "esm" output in patternfly, which bypasses any module configuration in tsconfig and makes the tests fail.
I don't think it's a good idea to bypass the tsconfig modules settings.

Reply:
To add a little more detail: importing is not possible, because the _patternfly-react_ library is not individually exporting those components. It makes them available only through the NotificationDrawer.* properties.

Reply:
OK, makes sense.  You cam only import something that is exported.